runtime/src/bank.rs

```diff
@@ -5090,6 +5090,7 @@ impl Bank {
                 let hash =
                     crate::accounts_db::AccountsDb::hash_account(self.slot(), &account, &pubkey);
                 rewrites_skipped.push((pubkey, hash));
+                assert_eq!(collected, CollectedInfo::default());
```

Comment:
How often does this case get hit? Does it make sense to have this as a `debug_assert!()` instead?

Reply:
are you worried about the cost?
I'd like it to be in here while we're stabilizing skipping rewrites.